Makefile

```diff
@@ -0,0 +1,115 @@
+# This Makefile should really be a bash script, but I don't know how to make
+# bash scripts that have multiple options, and since we're under deadline
+# pressure I've elected not to learn.
```

Comment:
probably a good thing to learn if you plan on using Linux for the rest of your life